conf/layer.conf

```diff
@@ -8,7 +8,22 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "celebrity-face-match"
 BBFILE_PATTERN_celebrity-face-match = "^${LAYERDIR}/"
 BBFILE_PRIORITY_celebrity-face-match = "24"
-LAYERSERIES_COMPAT_celebrity-face-match = "hardknott"
-LAYERDEPENDS_celebrity-face-match = "phytec ampliphy meta-ml"
+LAYERSERIES_COMPAT_celebrity-face-match = "scarthgap"
+LAYERDEPENDS_celebrity-face-match = "phytec ampliphy imx-machine-learning"
 
-DISTRO_VERSION_append = "-CelebrityFaceMatch-v0.6"
+BBMASK += "meta-ampliphy/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-imx/meta-bsp/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale-distro/recipes-graphics/wayland/weston-init.bbappend"
+
+DISTRO_VERSION:append = "-CelebrityFaceMatch-v0.7"
```

Comment:
#5 uses the v0.7 already. For quite some other changes, this seems not to be based on the kirkstone WIP?!

Reply:
All nine commits from your branch are included here. The branch is also up-to-date with yours. Could you point me to the missing changes?

Reply:
(Removed some commits, as mentioned above, but apart from that, this branch should be up-to-date.)

Reply:
I got it later, initially I was confused by the diff. Assumed you just branch the scarthgap of the (WIP/)kirkstone when we finished the PR ...

Reply:
Yes, this change is coming from your commit, which I cherry-picked/branched-off:
```
Author: Steffen Hemer <s.hemer@phytec.de>
Date:   Fri Jan 24 12:16:06 2025 +0100

    layer.conf: Set release version to 0.7

    Reviewed-by: Martin Schwan <m.schwan@phytec.de>
```
We can raise the version in a separate PR, when a new release is coming. For now, I just want to make the demo work on scarthgap/PD24.1.0.